README.md

```diff
@@ -8,42 +7,42 @@ current 5V power outputs.
 It also holds the internal On\|Off switch for the whole robot as well as
 the Start button which is used to start your robot code running.
 
-USB Interface
--------------
+## Instructions
 
-The Vendor ID is `1bda` (University of Southampton) and the product ID
-is `0010`.
+Using a posix system, you require `make`, the `arm-none-eabi` toolchain and `git`.
+Before attempting to build anything initialise all the submodules.
+```shell
+$ git submodule update --init --recursive
+```
+
+To build the main binary, run:
+```shell
+$ make
+```
+The binary will then be at `src/main.bin`.
+This will also build the library libopencm3 the first time you run it.
 
-The Power Board is controlled over USB by sending requests to the
-control endpoint.
-
-```python
-ctrl_transfer(
-    0x00,
-    64,
-    wValue=req_val,
-    wIndex=command.code,
-    data_or_wLength=req_data,
-)
+This can be flashed to an attached power board that has a bootloader using:
+```shell
+$ make -C src dfu
 ```
 
-| Parameter     | Value |
-|---------------|-------|
-| bmRequestType | 0x00  |
-| bRequest      | 64    |
+To build the bootloader, run:
+```shell
+$ make -C bootloader
+```
+The bootloader binary will then be at `bootloader/usb_dfu.bin`
 
-There are a list of ids defined in the firmware of the power board that
-will let you read and write values to it.
 
-It is recommended to read the source to further understand how to
-control this device.
+## USB Interface
 
-It should also be noted that as the control endpoint `0x00` is used to
-send data to this device, it is not actually compliant with the USB 2.0
-specification.
+The Vendor ID is `1bda` (University of Southampton) and the product ID
+is `0010`.
 
-udev Rule
----------
+The Power Board is controlled over USB serial, each command is its own line.
+The list of commands is TBC.
```

Comment:
Is the plan to add a list of finalised commands from the spec here?

Reply:
Added